pkg/cmd/server/kubernetes/master.go

```diff
@@ -133,3 +137,24 @@ func (c *MasterConfig) RunMinionController() {
 
 	glog.Infof("Started Kubernetes Minion Controller")
 }
+
+func (c *MasterConfig) createSchedulerConfig(configFactory *factory.ConfigFactory) (*scheduler.Config, error) {
```

Comment:
> The config creation requires the configFactory

Which is built from the `MasterConfig`: https://github.com/openshift/origin/pull/1420/files#diff-62e1d6e1ea7f763bd41c44733aa2fba2R104